src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -80,6 +82,9 @@
             <span class="invalid-feedback"
                   *ngIf="serviceForm.showError('service_id', frm, 'rgwPattern')"
                   i18n>The value does not match the pattern <strong>&lt;service_id&gt;[.&lt;realm_name&gt;.&lt;zone_name&gt;]</strong>.</span>
+            <span class="invalid-feedback"
+                  *ngIf="serviceForm.showError('service_id', frm, 'mdsPattern')"
+                  i18n>MDS service_id must start with a letter and contain alphanumeric characters or '.', '-', and '_'</span>
```

Comment:
```suggestion
                  i18n>MDS service id must start with a letter and contain alphanumeric characters or '.', '-', and '_'.</span>
```

Reply:
`service_id` is the name of the variable in our code, whenever we display messages to an user we should use regular language.